WalkingChallenge/Launch.storyboard

```diff
@@ -18,36 +18,44 @@
                         <viewControllerLayoutGuide type="bottom" id="xb3-aO-Qok"/>
                     </layoutGuides>
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <imageView userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="splash" translatesAutoresizingMaskIntoConstraints="NO" id="RhP-6k-bd8">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
+                            <imageView userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="252" verticalCompressionResistancePriority="749" image="Aga Khan Foundation LOGO_raster" translatesAutoresizingMaskIntoConstraints="NO" id="IU1-63-VQT">
+                                <rect key="frame" x="44" y="160" width="327" height="179"/>
```

Comment:
Why are the bounds of the rect being adjusted?

Reply:
More constraints adjustments. Adjustments were made to keep the logo consistent in all screens. I may have approached this incorrectly, any advise would be great.